AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -600,6 +598,11 @@ extension MeetingModel: ActiveSpeakerObserver {
     }
 
     func activeSpeakerDidDetect(attendeeInfo: [AttendeeInfo]) {
+        videoModel.updateRemoteVideoStatesBasedOnActiveSpeakers(activeSpeakers: attendeeInfo)
+        if activeMode == .video {
+            videoModel.videoUpdatedHandler?()
+        }
```

Comment:
I think we have to call `updateRemoteVideoStatesBasedOnActiveSpeakers` when `activeMode != .video` as well. Otherwise, if user was viewing other tabs (e.g. roster) and then switch back to the video tab, the video tiles may not be updated according to active speaker logic, because `activeSpeakerDidDetect()` may not be called again after user switched to the video tab.